eureka_ml_insights/data_utils/__init__.py

```diff
@@ -68,8 +68,7 @@
     ASTEvalTransform,
     PrependStringTransform,
     ExtractAnswerGrid,
-    ExtractAnswerSpatialMap,
-    ExtractAnswerMaze,
+    ExtractAnswerSpatialMapAndMaze,
```

Comment:
Please add "ExtractQuestionOptions" so that when we run the formatters this import does not get removed (considered unused).

Reply:
ahh, yes will do

Reply:
done and checked in